R/guide-colorbar.r

```diff
@@ -567,6 +567,57 @@ guide_gengrob.colorbar <- function(guide, theme) {
 #' @rdname guide_colourbar
 guide_colorbar <- guide_colourbar
 
+#' @export
+#' @rdname guide_colourbar
+#'
+#' @param even.steps Should the bin sizes reflect their size or be even across all
```

Comment:
I don't quite follow what this means, or why you'd want it to be false.

Reply:
If you set the breaks to be uneven intervals, e.g. [5, 10, 20, 40] should this be reflected in the relative size of each bin in the colour ramp

Reply:
Do you mean: Should the rendered size of the bins be proportional to their length in the data space?

Reply:
yeah - is that a clearer phrasing?

Reply:
Yeah, I think so